internal/tools/aws/database.go

```diff
@@ -183,7 +184,20 @@ func (d *RDSDatabase) rdsDatabaseProvision(installationID string, logger log.Fie
 		return err
 	}
 
-	err = d.client.rdsEnsureDBClusterCreated(awsID, *vpcs[0].VpcId, rdsSecret.MasterUsername, rdsSecret.MasterPassword, logger)
+	encryptionKey, err := d.client.kmsCreateSymmetricKeyWithAlias(awsID, KMSAliasNameRDS(awsID), "Key used for encrypting RDS database")
+	if err != nil && !IsErrorCode(err, kms.ErrCodeAlreadyExistsException) {
+		// If key is created but alias did not, this key should be scheduled for deletion.
+		if encryptionKey != nil {
+			deletionKeyErr := d.client.kmsScheduleKeyDeletion(*encryptionKey.KeyId, 7)
+			if deletionKeyErr != nil {
+				logger.WithError(deletionKeyErr).Warnf("Failed to schedule encryption key %s for deletition", *encryptionKey.KeyId)
+			}
+		}
+
+		return errors.Wrapf(err, "unable provision RDS database for installation %s", installationID)
```

Comment:
This error should probably be a bit more specific like:
```suggestion
		return errors.Wrapf(err, "unable to create RDS encryption key for installation %s", installationID)
```